assets/images/ic_error.svg

```diff
@@ -0,0 +1,35 @@
+<!-- LinShare is an open source filesharing software, part of the LinPKI software
```

Comment:
We should naming it more detail. like `ic_unexpected_error`. There might be more error icons in the future

Reply:
You're right, done